src/netbase.cpp

```diff
@@ -788,15 +788,15 @@ bool CNetAddr::IsTor() const
 bool CNetAddr::IsLocal() const
 {
     // IPv4 loopback
-   if (IsIPv4() && (GetByte(3) == 127 || GetByte(3) == 0))
-       return true;
+    if (IsIPv4() && (GetByte(3) == 127 || GetByte(3) == 0))
+        return true;
```

Comment:
Okay. I see. You are right that the formatting is wrong there. But the diff you create can be hard to read for others. At least it would be good to separate the style changes from the actual code changes. Also i wouldn't care to much about existing inconsistency in code formatting because we could run one `clang-format` over everything (see https://github.com/bitcoin/bitcoin/pull/5387).